analyses/cms-open-data-ttbar/ml.py

```diff
@@ -148,24 +138,24 @@ def define_features(df: ROOT.RDataFrame) -> ROOT.RDataFrame:
         """,
     )
 
-
 def predict_proba(df: ROOT.RDataFrame) -> ROOT.RDataFrame:
     """get probability scores for every permutation in event"""
 
     # in inference, odd model applied to even events, while even model to odd events
     # read more about inference in dedicated part of AGC documentation:
     # https://agc.readthedocs.io/en/latest/taskbackground.html#machine-learning-component
+    
+    models_to_rootfiles()
```

Comment:
We could just decide to remove the need for this conversion step and just save the .root files directly. The conversion doesn't add anything relevant to the educational purposes of the analysis. Maybe it's not super important though